### DISCUSSION TITLE
v5.0.0-alpha.3

### DISCUSSION THREAD

**Discussion by joschi** [dropwizard/dropwizard#9405] [Announcements]
## What's Changed
* chore(deps): update maven plugins (release/5.0.x) (minor) by @renovate in https://github.com/dropwizard/dropwizard/pull/9242
* fix(deps): update mockito monorepo to v5.13.0 (release/5.0.x) (minor) by @renovate in https://github.com/dropwizard/dropwizard/pull/9243
* chore(deps): update github/codeql-action action to v3.26.6 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9251
* fix(deps): update dependency net.bytebuddy:byte-buddy to v1.15.1 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9252
* chore(deps): update error_prone.version to v2.31.0 (release/5.0.x) (minor) by @renovate in https://github.com/dropwizard/dropwizard/pull/9253
* fix(deps): update dependency org.apache.commons:commons-lang3 to v3.17.0 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9256
* Use SLF4J BOM to manage SLF4J versions by @joschi in https://github.com/dropwizard/dropwizard/pull/9261
* chore(deps): update actions/upload-artifact action to v4.4.0 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9262
* chore(deps): update dependency sphinx-autobuild to v2024.9.3 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9267
* fix(deps): update dependency org.liquibase:liquibase-core to v4.29.2 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9272
* fix(deps): update dependency org.checkerframework:checker-qual to v3.47.0 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9273
* fix(deps): update jetty.version to v12.0.13 (release/5.0.x) (patch) by @renovate in https://github.com/dropwizard/dropwizard/pull/9274
* chore(deps): update dependency org.apache.maven.shared:maven-filtering to v3.4.0 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9280
* chore(deps): update dependency jp.skypencil.errorprone.slf4j:errorprone-slf4j to v0.1.28 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9286
* fix(deps): update dependency org.jboss.logging:jboss-logging to v3.6.1.final (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9287
* chore(deps): update actions/setup-java action to v4.3.0 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9288
* fix(deps): update dependency org.apache.tomcat:tomcat-jdbc to v10.1.29 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9291
* chore(deps): update error_prone.version to v2.32.0 (release/5.0.x) (minor) by @renovate in https://github.com/dropwizard/dropwizard/pull/9301
* fix(deps): update dependency net.java.dev.jna:jna to v5.15.0 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9302
* fix(deps): update dependency org.apache.httpcomponents.core5:httpcore5 to v5.3 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9303
* chore(deps): update dependency org.apache.maven.plugins:maven-gpg-plugin to v3.2.6 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9306
* chore(deps): update github/codeql-action action to v3.26.7 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9307
* chore(deps): update dependency com.uber.nullaway:nullaway to v0.11.3 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9316
* chore(deps): update dependency sphinx-autobuild to v2024.9.17 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9317
* fix(deps): update dependency org.apache.tomcat:tomcat-jdbc to v10.1.30 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9318
* chore(deps): update dependency sphinx-autobuild to v2024.9.19 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9326
* chore(deps): update github/codeql-action action to v3.26.8 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9327
* chore(deps): update github/codeql-action action to v3.26.9 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9337
* fix(deps): update guava monorepo to v33.3.1-jre (release/5.0.x) (patch) by @renovate in https://github.com/dropwizard/dropwizard/pull/9338
* chore(deps): update github actions (release/5.0.x) (minor) by @renovate in https://github.com/dropwizard/dropwizard/pull/9339
* fix(deps): update dependency org.hibernate.orm:hibernate-core to v6.6.1.final (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9319
* fix(deps): update dependency org.apache.httpcomponents.client5:httpclient5 to v5.4 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9330
* chore(deps): update dependency org.apache.maven.plugins:maven-gpg-plugin to v3.2.7 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9357
* chore(deps): update dependency org.cyclonedx:cyclonedx-maven-plugin to v2.8.2 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9358
* chore(deps): update junit5 monorepo to v5.11.1 (release/5.0.x) (patch) by @renovate in https://github.com/dropwizard/dropwizard/pull/9359
* fix(deps): update dependency net.bytebuddy:byte-buddy to v1.15.3 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9360
* chore(deps): update dependency org.apache.maven.plugins:maven-archetype-plugin to v3.3.0 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9361
* fix(deps): update mockito monorepo to v5.14.0 (release/5.0.x) (minor) by @renovate in https://github.com/dropwizard/dropwizard/pull/9363
* chore(deps): update dependency org.apache.maven.plugins:maven-javadoc-plugin to v3.10.1 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9378
* fix(deps): update dependency org.testcontainers:testcontainers-bom to v1.20.2 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9380
* chore(deps): update github/codeql-action action to v3.26.10 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9379
* fix(deps): update jetty.version to v12.0.14 (release/5.0.x) (patch) by @renovate in https://github.com/dropwizard/dropwizard/pull/9381
* fix(deps): update mockito monorepo to v5.14.1 (release/5.0.x) (patch) by @renovate in https://github.com/dropwizard/dropwizard/pull/9382
* fix(deps): update dependency io.dropwizard.metrics:metrics-bom to v4.2.28 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9388
* chore(deps): update dependency sphinx-autobuild to v2024.10.3 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9383
* chore(deps): update error_prone.version to v2.33.0 (release/5.0.x) (minor) by @renovate in https://github.com/dropwizard/dropwizard/pull/9384
* fix(deps): update dependency org.checkerframework:checker-qual to v3.48.0 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9385
* chore(deps): update github/codeql-action action to v3.26.11 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9391
* chore(deps): update junit5 monorepo to v5.11.2 (release/5.0.x) (patch) by @renovate in https://github.com/dropwizard/dropwizard/pull/9396
* chore(deps): update actions/cache action to v4.1.0 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9397
* fix(deps): update dependency com.fasterxml.jackson:jackson-bom to v2.18.0 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9362
* chore(deps): update dependency org.apache.maven.plugins:maven-surefire-plugin to v3.5.1 (release/5.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9400
* Use platform threads instead of virtual threads for thread pools by @joschi in https://github.com/dropwizard/dropwizard/pull/9402

**Full Changelog**: https://github.com/dropwizard/dropwizard/compare/v5.0.0-alpha.2...v5.0.0-alpha.3

<hr /><em>This discussion was created from the release <a href='https://github.com/dropwizard/dropwizard/releases/tag/v5.0.0-alpha.3'>v5.0.0-alpha.3</a>.</em>

**Comment by nabeelq** [comment]
@joschi what is the expected release date for version 5.0.0?

**Reply by joschi** [reply]
There is no release schedule.
